GetPlayersInSegmentSample/get_players_in_segment_api_sample.js

```diff
@@ -0,0 +1,64 @@
+handlers.GetPlayersInSegmentSample = function (args, context) {
+
+    /*
+        Sample code to use the GetPlayersInSegment API to process the player profiles in a Segment.
+        The GetPlayersInSegment API pages through the all the player profiles
+        in the Segment in batches of size 'MaxBatchSize'.
+        API Doc: https://learn.microsoft.com/en-us/rest/api/playfab/server/play-stream/get-players-in-segment?view=playfab-rest
+    */
+
+    var request =  {
+     GetProfilesAsync: true,  // setting to true is highly recommended to avoid network timeouts
+     MaxBatchSize: 1000,      // 1000 is the default value. Maximum is 10,000
+     SegmentId: "AAAAAAAAA"  // provide your SegmentId here
+    }
+
+    // make the first GetPlayersInSegment API call
+    var playersInSegmentResult =  server.GetPlayersInSegment(request);
+    
+    // process until continuation token is not null to get all the profiles in this Segment
+    while (playersInSegmentResult.ContinuationToken != null) 
+    {
+        // get the current batch of player profiles
+        var playerProfiles = playersInSegmentResult.PlayerProfiles;
+
+        if (playerProfiles && playerProfiles.length > 0)
+        {
+            // TODO: 
+            // Add your logic here to process the above playerProfiles
+        }
+
+        // get the next batch of player profiles in the Segment
+        request =  {
+            ContinuationToken: playersInSegmentResult.ContinuationToken,
+            GetProfilesAsync: true,
+            MaxBatchSize: 1000,
+            SegmentId: "AAAAAAAAA" // provide your SegmentId here
+        }
```

Comment:
You can use the same request object that was created above, and just set the ContinuationToken value in it.

request.ContinuationToken = playersInSegmentResult.ContinuationToken;

Then there is only one place where the other request options need to be set.

Reply:
Done